### PR TITLE
[5.3] Bring the Authenticate middleware into core

### PR DESCRIPTION
**Pull request by JosephSilber**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Seems pretty useful to me. It's always nice to cut down on boiler plate files from small apps.

:+1:

File: src/Illuminate/Auth/Middleware/Authenticate.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Illuminate\Foundation\Http;
```
Comment (taylorotwell) [inline src/Illuminate/Auth/Middleware/Authenticate.php]: Namespace is wrong here.

File: src/Illuminate/Auth/Middleware/Authenticate.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+use Closure;
+use Illuminate\Auth\AuthenticationException;
```
Comment (taylorotwell) [inline src/Illuminate/Auth/Middleware/Authenticate.php]: This import actually not needed once we're in Auth namespace.

Comment (GrahamCampbell) [inline src/Illuminate/Auth/Middleware/Authenticate.php]: once you fix the namespace, styleci should actually remove this import automatically for us ;)

**Pull request merged**